connectors/src/connectors/google_drive/index.ts

```diff
@@ -680,7 +689,7 @@ export class GoogleDriveConnectorManager extends BaseConnectorManager<null> {
         s.driveFileId,
         s.driveSheetId
       ),
-      parentInternalId: s.driveFileId,
+      parentInternalId: getDocumentId(s.driveFileId),
```

Comment:
nit: wasn't there a comment about always using internalId in connectors and keeping documentId for core by spolu in a former PR?

Reply:
indeed, I was aiming for the lowest possible diff size here :)
changing it